string url
---
string task_announce
string robot_announce
string object_announce
string user_announce
string place_announce
string data
int32 task_id
int32 robot_id
int32 object_id
int32 user_id
int32 place_id
